CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - **CUMULUS-2312** - RDS Migration Epic Phase 3
+  - **CUMULUS-2645**
+    - Removed dynamo structural migrations and related code from `@cumulus/api`
+    - Removed unused index functionality for all tables other than
+      `ReconciliationReportsTable` from `dbIndexer` lambda
+    - Removed `executeMigrations` lambda
+    - Removed `granuleFilesCacheUpdater` lambda
+    - Removes dynamo files table from `data-persistence` module.  *This table and
```

Comment:
```suggestion
    - Removed dynamo files table from `data-persistence` module.  *This table and
```
This is a nit but the suggestion is to maintain consistency with the other changelog notes